USER_MANUAL.md

```diff
@@ -3042,6 +3042,76 @@ Using `rbreak <regex>` you can set breakpoints on multiple functions.
 -   `(gdb) break definition.kore:break -> No source file named definition.kore.`
 send `-ccopt -g` to kompile in order to generate debug info symbols.
 
+Profiling your K semantics
+--------------------------
+
+### General instructions
+
+1. The first thing to be aware of is in order to get meaningful data,
+you need to build the semantics and all of its dependencies with
+optimizations enabled but _without the frame pointer elimination
+optimization_. For example, for EVM, this means rebuilding GMP, MPFR,
+JEMalloc, Crypto++, SECP256K1, etc with the following `exports`.
+```shell export CFLAGS="-g -O2 -fno-omit-frame-pointer"; export
+CXXFLAGS="-DNDEBUG -O2 -fno-omit-frame-pointer" ```
+
```

Comment:
@ChristianoBraga the formatting got messed up here, I guess you called `format` to do the line wrapping.

Please make sure to review the changes yourself (self-review) in browser.